### PR TITLE
Add support for efi driver for ext fs

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by Itxaka** [comment]
tests will fail here until the package is updated with those new file drivers

**Review by mauromorales — APPROVED** [review]

File: pkg/action/build-uki.go [hunk 1]
```diff
@@ -630,13 +655,15 @@ func (b *BuildUKIAction) imageFiles(sourceDir string) (map[string][]string, erro
...
-		"EFI/BOOT":       {filepath.Join(sourceDir, "BOOTX64.EFI")},
-		"EFI/kairos":     {},
-		"EFI/tools":      {},
-		"loader":         {filepath.Join(sourceDir, "loader.conf")},
-		"loader/entries": {},
-		"loader/keys":    {},
+		"EFI":                 {},
+		"EFI/BOOT":            {filepath.Join(sourceDir, "BOOTX64.EFI")},
+		"EFI/systemd":         {},
+		"EFI/systemd/drivers": {filepath.Join(sourceDir, "ext2_x64.efi")},
```
Comment (jimmykarily) [inline pkg/action/build-uki.go:661]: what about the arm one?

Comment (Itxaka) [reply pkg/action/build-uki.go:661]: no arm support for now until we can build it ourselves. Also the BOOTX64.efi file points to no arm support as well, lol

**Comment by jimmykarily** [comment]
tests broke

**Comment by Itxaka** [comment]
> tests broke

comment above:

> tests will fail here until the package is updated with those new file drivers

**Pull request closed without merging**